include/seqan3/io/sequence_file/format_fasta.hpp

```diff
@@ -339,23 +343,55 @@ class format_fasta
               typename seq_type>
     void write_seq(stream_it_t & stream_it,
                    sequence_file_output_options const & options,
-                   seq_type && seq)
+                   seq_type & seq)
```

Comment:
why don't we take a temporary here anymore?

Reply:
Hm, it's debatable what the correct thing here is. My motivation in changing it was that these functions are not being forwarded to, so they never receive temporaries. But the general policy was that we always use `type &&` for out-parameters with type deduction so we might as well keep it.